capstone/capdb/tasks.py

```diff
@@ -430,13 +432,14 @@ def extract_citations_per_vol(self, volume_id):
 
         ExtractedCitation.objects.bulk_create([ExtractedCitation(
             cite_original=c["cite_original"],
+            normalized_cite=normalize_cite(c["cite_original"]),
             cited_by=c["cited_by"],
             reporter_name_original=c["reporter_name_original"],
             volume_number_original=c["volume_number_original"],
             page_number_original=c["page_number_original"]) for c in extracted_citations])
 
-    fieldnames = ['case_origin', 'missed_cites_per_case', 'missed_cites']
-    with open("/tmp/missed_citations.csv", "a+") as f:
-        writer = csv.DictWriter(f, fieldnames=fieldnames)
+    with open("/tmp/missed_citations-%s.csv" % self.request.id, "w+") as f:
```

Comment:
Let's put this in a subfolder like `/tmp/missed_citations` in case anyone needs to find anything in the /tmp dir while it's running. Can throw in a line like `Path("/tmp/missed_citations").mkdir(exist_ok=True)` first.